figures/label_sampling_experiment/plot_scripts/label_sampling_gen_figure_generator.py

```diff
@@ -75,6 +76,8 @@ def plot_performance_graph(
 
             # remove x axis labels
             axes[row_ind][col_ind].set_xlabel('')
+            if metric == "AUROC":
+                axes[row_ind][col_ind].set_ylim([0.5, 1])
```

Comment:
Looks like you still see your same trends, but some are a bit more flat..I don't think any interpretations changed though?

Reply:
Yeah change the axis makes the trend be more flat, but the interpretation remains the same.